ivy.el

```diff
@@ -973,16 +974,19 @@ If the text hasn't changed as a result, forward to `ivy-alt-done'."
 (defvar ivy-completion-end nil
   "Completion bounds end.")
 
-(defun ivy-immediate-done ()
-  "Exit the minibuffer with current input instead of current candidate."
-  (interactive)
+(defun ivy-immediate-done (&optional arg)
+  "Exit the minibuffer with current input instead of current candidate.
+When ARG is t, exit with empty input."
```

Comment:
Nitpick: "When ARG is t..." is not entirely accurate, since `(interactive "P")` does not set `ARG` to `t`. It would be more accurate and conventional to say "When ARG is non-nil..."